website/pages/en/resources.js

```diff
@@ -89,36 +74,77 @@ class Resources extends React.Component {
     const VideoCardList = () => (
       <Section background="tint">
         <div className="content">
-          <MarkdownBlock>{textContent.introtext}</MarkdownBlock>
+          <h1 style={{marginTop: -35}}>Resources</h1>
+          <p>The follow is a list of resources related to React Native for Windows and macOS to help you get started or to better accelerate your development in this area.</p>
```

Comment:
Feels like this page is more highlighting the various offerings from Microsoft in the React Native space rather than specifically being about resources (like docs, videos, guides etc.,) and they are not specific to RN for Windows/Mac only either. So, maybe change header/title/description to reflect that a bit more? Something like - "The following is a suite of offerings from Microsoft in the React Native space to help you target Desktop platforms as well as accelerate your development efficiency with React Native".

Reply:
Hmm, sidebars are only enabled for markdown-only files. I can remove the styling/cards here and create a separate markdown file - then the resources will be versioned too, would that work? Only drawback would be that everything would then go into a markdown table which is a bit less readable...

Reply:
I really like the styling cards, is there a way to just embed these as html blocks inside a markdown file so we can preserve the format? We do similar things in the landing pages of the repos to embed the top header items as html content.